FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/JobManager.cs

```diff
@@ -43,48 +43,23 @@ public class JobManager : IDisposable, IAsyncDisposable
             _jobExecutor = jobExecutor;
             _fhirSpecificationProvider = fhirSpecificationProvider;
             _jobConfiguration = jobConfiguration.Value;
+
             _logger = logger;
         }
 
         /// <summary>
-        /// Resume an active job or trigger new job from job store
+        /// Resume an active job or trigger new job from job store.
         /// and execute the job.
         /// </summary>
         /// <param name="cancellationToken">cancellation token.</param>
         /// <returns>Completed task.</returns>
         public async Task RunAsync(CancellationToken cancellationToken = default)
         {
-            // Acquire lock to ensure the job store is changed from only one client.
-            var lockResult = await _jobStore.AcquireJobLock(cancellationToken);
-            if (!lockResult)
+            // Acquire an active job from the job store.
+            var job = await _jobStore.AcquireJobAsync(cancellationToken);
```

Comment:
Maybe renaming the function to `AcquireActiveJobAsync()` will be more accurate?

Reply:
Sounds good. Updated~